velox/exec/tests/AggregationTest.cpp

```diff
@@ -1430,6 +1430,20 @@ TEST_F(AggregationTest, groupingSetsEmptyInput) {
       plan,
       "SELECT count(c2) as count_c2 FROM tmp WHERE c1 < 0 GROUP BY GROUPING SETS ((c1), ())");
 
+  // No aggregations with GROUPING SETS.
+  plan = PlanBuilder()
+             .values({data})
+             .filter("c1 < 0")
+             .groupId({"c1"}, {{"c1"}, {}}, {})
+             .partialAggregation({"c1", "group_id"}, {}, {})
+             .finalAggregation()
+             .project({"c1"})
```

Comment:
I wonder if it would be possible to verify the 'groupId' column produced by this aggregation.

Reply:
Modified one of the test.